lib/galaxy/config/config_manage.py

```diff
@@ -12,6 +12,7 @@
 import requests
 import yaml
 from boltons.iterutils import remap
+from gravity.util import settings_to_sample
```

Comment:
I guess that it won't be possible for gravity to depend on `galaxy-config` in the future, unless we refactor this.

Reply:
We could make it a local import, or publish the config schema separately, but I thought we can do this once we're trying to do this.

Reply:
This broke the `test_galaxy_packages` tests on CircleCI, I guess because `gravity` needs to be added to `packages/config/requirements.txt` .